trin-core/src/portalnet/overlay.rs

```diff
@@ -160,7 +159,12 @@ impl OverlayProtocol {
             .collect()
     }
 
-    pub async fn send_ping(&self, data_radius: U256, enr: Enr) -> Result<Vec<u8>, String> {
+    pub async fn send_ping(
+        &self,
+        data_radius: U256,
+        enr: Enr,
+        protocol: String,
```

Comment:
Any thoughts about updating the `protocol` type here (and the other functions in this module) to an enum of the available network constants? Using a string seems unnecessary and ambiguous